nextjs-13/src/lib/handler/base.handler.ts

```diff
@@ -0,0 +1,37 @@
+import { User } from '@prisma/client';
+import { NextResponse, NextRequest } from 'next/server';
+import { getToken } from 'next-auth/jwt';
+
+import { findUserById } from 'repositories/user.repository';
+
+export default async function baseHandler(
```

Comment:
`base` would mean that it could be extended (with more features). How about naming it `appHandler` or `defaultHandler` instead? In addition, since we are checking for user authentication, it could be named `authRequiredHandler`.

Reply:
I see. Yes, these would be better names, let me update 👀

Reply:
Done! [And this is the new commit](https://github.com/nimblehq/js-framework-benchmark/pull/96/commits/455c612b291d7e0f3ddf58b4f2b24ae0f3e08756#diff-10b91e667909ffe6c8b2d34d735e0fa46be166ba37cf1b5ec24f7ea6b54658bbR8)